shell/browser/api/electron_api_crash_reporter.cc

```diff
@@ -16,6 +16,7 @@
 #include "base/path_service.h"
 #include "base/strings/utf_string_conversions.h"
 #include "base/threading/thread_restrictions.h"
+#include "base/trace_event/trace_event.h"
```

Comment:
@RaisinTen Out of curiosity,
Noticed that `TRACE_EVENT0` is defined in base/trace_event/common/trace_event_common.h
But there is a different file included, turning it into a transitive include.
Any reason behind that?

Reply:
@bavulapati it's because of https://source.chromium.org/chromium/chromium/src/+/main:base/trace_event/common/trace_event_common.h;l=286-290;bpv=1;bpt=0;drc=38e4ebbecc58a2a6a8637dbebfd221df3b4b8273:
```cc
// Check that nobody includes this file directly.  Clients are supposed to
// include the surrounding "trace_event.h" of their project instead.
#if defined(TRACE_EVENT0)
#error "Another copy of this file has already been included."
#endif
```